packages/mermaid/src/diagrams/flowchart/parser/flow-lines.spec.js

```diff
@@ -13,75 +13,134 @@ describe('[Lines] when parsing', () => {
+    flow.parser.parse('graph TD\n' + 'A-->B\n' + 'linkStyle default interpolate basis');
 
-    const vert = flow.parser.yy.getVertices();
     const edges = flow.parser.yy.getEdges();
 
     expect(edges.defaultInterpolate).toBe('basis');
   });
 
   it('should handle line interpolation numbered definitions', function () {
-    const res = flow.parser.parse(
+    flow.parser.parse(
       'graph TD\n' +
         'A-->B\n' +
         'A-->C\n' +
         'linkStyle 0 interpolate basis\n' +
         'linkStyle 1 interpolate cardinal'
     );
 
-    const vert = flow.parser.yy.getVertices();
     const edges = flow.parser.yy.getEdges();
 
     expect(edges[0].interpolate).toBe('basis');
     expect(edges[1].interpolate).toBe('cardinal');
   });
 
-  it('should handle line interpolation multi-numbered definitions', function () {
-    const res = flow.parser.parse(
-      'graph TD\n' + 'A-->B\n' + 'A-->C\n' + 'linkStyle 0,1 interpolate basis'
+  it('should handle line interpolation non-numbered definitions', function () {
+    flow.parser.parse(
+      'graph TD\n' +
+        'A-->B\n' +
+        'linkStyle - interpolate basis\n' +
+        'A-->C\n' +
+        'linkStyle - interpolate cardinal'
     );
 
-    const vert = flow.parser.yy.getVertices();
+    const edges = flow.parser.yy.getEdges();
+
+    expect(edges[0].interpolate).toBe('basis');
+    expect(edges[1].interpolate).toBe('cardinal');
+  });
+
+  it('should handle line interpolation multi-numbered definitions', function () {
+    flow.parser.parse('graph TD\n' + 'A-->B\n' + 'A-->C\n' + 'linkStyle 0,1 interpolate basis');
+
     const edges = flow.parser.yy.getEdges();
 
     expect(edges[0].interpolate).toBe('basis');
     expect(edges[1].interpolate).toBe('basis');
   });
 
   it('should handle line interpolation default with style', function () {
-    const res = flow.parser.parse(
+    flow.parser.parse(
       'graph TD\n' + 'A-->B\n' + 'linkStyle default interpolate basis stroke-width:1px;'
     );
 
-    const vert = flow.parser.yy.getVertices();
     const edges = flow.parser.yy.getEdges();
 
     expect(edges.defaultInterpolate).toBe('basis');
   });
 
   it('should handle line interpolation numbered with style', function () {
-    const res = flow.parser.parse(
+    flow.parser.parse(
       'graph TD\n' +
         'A-->B\n' +
         'A-->C\n' +
         'linkStyle 0 interpolate basis stroke-width:1px;\n' +
         'linkStyle 1 interpolate cardinal stroke-width:1px;'
     );
 
-    const vert = flow.parser.yy.getVertices();
     const edges = flow.parser.yy.getEdges();
 
     expect(edges[0].interpolate).toBe('basis');
     expect(edges[1].interpolate).toBe('cardinal');
   });
 
+  it('should handle line interpolation non-numbered with style', function () {
+    flow.parser.parse(
+      'graph TD\n' +
+        'A-->B\n' +
+        'linkStyle - interpolate basis stroke-width:1px;\n' +
```

Comment:
Can we apply it twice?
```
linkStyle -
linkStyle -
```
I have not checked whether it should work or not, I am not well aware of flowchart behavior. Could you check it for me?

Reply:
Now you can actually play around with PR Version of mermaid in the deploy previews. 

It was actually added when I had similar questions on a PR and it was easier to test out all the edge cases live instead of either asking the author or checking out and building it locally.

Reply:
The answer is yes, we can apply linkStyle twice and it overrides the previous one. It does not merge attributes. Resolving this, because linkStyle with index has the same behavior.